### ISSUE TITLE
Rails 5.1 support

### ISSUE THREAD

**Issue by dwyerb64** [actmd/abraham#12]
Hi @jabbett 
Tried out this gem today and blown away (really impressed).
I forked it and changed the rails dependency for it to work with rails 5.1:

> See abraham.gemspec line 20.

`s.add_dependency 'rails', '~> 5.0', '>= 5.0.0.1'`

**Comment by jabbett** [comment]
I'll try to push a 1.3 release on Monday with this change.

**Comment by seanfcarroll** [comment]
Hi it doesn't seem to be working...

```
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 5.1.4)

  In Gemfile:
    rails (~> 5.1)

    abraham was resolved to 1.2, which depends on
      rails (>= 5.0.0.1, ~> 5.0.0)

```

**Comment by Velora** [comment]
This is a great gem. I wanted to leave a reminder that it looks like this small fix for Rails 5.1 compatibility wasn't released yet.

**Comment by markhallen** [comment]
This problem is still there:

    Bundler could not find compatible versions for gem rails:
    In Gemfile:
      rails (~> 5.1.4)

      abraham was resolved to 1.2, which depends on
        rails (>= 5.0.0.1, ~> 5.0.0)

**Comment by markhallen** [comment]
Thanks @jabbett. I integrated this into my project and it is a great gem: thanks a lot.
